### PR TITLE
Add WABT

### PR DESCRIPTION
**Pull request by kpine**
WABT: The WebAssembly Binary Toolkit

### CODE REVIEW

File: bucket/wabt.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/WebAssembly/wabt/releases/download/$version/wabt-$version-win64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/WebAssembly/wabt/releases/download/$version/wabt-$version-win32.zip"
+            }
+        }
+    }
```
Comment (Ash258) [inline bucket/wabt.json]:
```suggestion
    },
    "hash": {
        "url": "$url.sha256"
    }
```

File: bucket/wabt.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+    "homepage": "https://github.com/WebAssembly/wabt",
+    "description": "WABT (we pronounce it \"wabbit\") is a suite of tools for WebAssembly.",
```
Comment (Ash258) [inline bucket/wabt.json]: Use single quote. No need to escape them.

```suggestion
    "description": "WABT (we pronounce it 'wabbit') is a suite of tools for WebAssembly.",
```

File: bucket/wabt.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        "wasm-opcodecnt.exe",
+        "wasm-strip.exe",
+        "wasm-validate.exe",
+        "wasm2c.exe",
+        "wasm2wat.exe",
+        "wast2json.exe",
+        "wat-desugar.exe",
+        "wat2wasm.exe"
+    ],
+    "checkver": {
```
Comment (Ash258) [inline bucket/wabt.json]:
```suggestion
    "checkver": "github"
```

File: bucket/wabt.json (hunk 2 above)
Comment (kpine) [reply bucket/wabt.json]: Done.

**Comment by kpine** [comment]
Updated based on comments.

Use autoupdate hash to re-generate new manifest for 1.0.8 release (which added the sha files).

I can squash these commits if you want, wasn't sure what's preferred here.

**Comment by kotx** [comment]
Why is this in extras and not the main bucket?

**Pull request merged**